web/react/components/search_results.jsx

```diff
@@ -64,6 +65,11 @@ SearchItem = React.createClass({
                 dispatchError(err, "getPost");
             }
         );
+
+    var postChannel = ChannelStore.get(this.props.post.channel_id)
+    var teammate = postChannel.type === 'D' ? UserStore.getProfile(postChannel.name.substring(postChannel.name.indexOf('_')+2)).username : "";
```

Comment:
I thought there is a method to handle the splitting of ids out of the channel name?  this looks like the channel name could be your name vs the other username?